codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -164,4 +183,51 @@ class FileCodesignVisitor {
       processManager: processManager,
     );
   }
+
+  /// Visit and codesign a binary with / without entilement.
+  ///
+  /// At this stage, the virtual [entitlementCurrentPath] accumulated through the recursive visit, is compared
+  /// against the paths extracted from [fileWithEntitlements], to help determine if this file should be signed
+  /// with entitlements.
+  Future<void> visitBinaryFile({required FileSystemEntity binaryEntity, required String entitlementParentPath}) async {
+    if (await isSymlink(binaryEntity.absolute.path, processManager)) {
+      return;
+    }
+
+    final String currentFileName = binaryEntity.absolute.path.split('/').last;
+    final String entitlementCurrentPath = '$entitlementParentPath/$currentFileName';
+
+    if (!fileWithEntitlements.contains(entitlementCurrentPath) &&
+        !fileWithoutEntitlements.contains(entitlementCurrentPath)) {
+      log.severe('The system has detected a binary file at $entitlementCurrentPath\n'
+          'but it is not in the entitlements configuartion files your provided \n'
```

Comment:
```suggestion
          'but it is not in the entitlements configuration files you provided \n'
```

Reply:
Actually, I suspect logging just the path is enough, the rest of this message seems redundant with `fixItInstructions`.

Reply:
Thanks for correcting!